frontend/src/pages/projects/dataConnections/const.ts

```diff
@@ -9,10 +10,35 @@ export enum AWS_KEYS {
   AWS_S3_BUCKET = 'AWS_S3_BUCKET',
 }
 
-export const AWS_REQUIRED_KEYS: AWS_KEYS[] = [
-  AWS_KEYS.NAME,
-  AWS_KEYS.ACCESS_KEY_ID,
-  AWS_KEYS.SECRET_ACCESS_KEY,
+export const AWS_FIELDS: FieldOptions[] = [
+  {
+    key: AWS_KEYS.NAME,
+    label: AWS_KEYS.NAME,
+    isRequired: true,
+  },
+  {
+    key: AWS_KEYS.ACCESS_KEY_ID,
+    label: AWS_KEYS.ACCESS_KEY_ID,
+    isRequired: true,
+  },
+  {
+    key: AWS_KEYS.SECRET_ACCESS_KEY,
+    label: AWS_KEYS.SECRET_ACCESS_KEY,
+    isPassword: true,
+    isRequired: true,
+  },
+  {
+    key: AWS_KEYS.AWS_S3_BUCKET,
+    label: AWS_KEYS.AWS_S3_BUCKET,
+  },
+  {
+    key: AWS_KEYS.S3_ENDPOINT,
+    label: AWS_KEYS.S3_ENDPOINT,
+  },
```

Comment:
Order changed... don't think that is desired or expected(?)... probably want to revert the order change.

Yours:
![image](https://user-images.githubusercontent.com/8126518/235773466-677824d2-03b7-4790-a7d2-322123c2f030.png)

Originally:
![image](https://user-images.githubusercontent.com/8126518/235773535-52d43739-4760-40e3-a1e7-833f926c3a42.png)

Reply:
++